app/views/about/index.html.erb

```diff
@@ -70,3 +156,28 @@
   }
   getContributors('https://api.github.com/repos/CircuitVerse/CircuitVerse/contributors');
 </script>
+<script>
```

Comment:
No need of slides, just have a static page

Reply:
And what about the overall design of the core team, mentor, alumni card? Is it good or, should the design change?

Reply:
Please include full screenshot , cc @nitin10s

Reply:
![04 09 2021_15 21 46_REC](https://user-images.githubusercontent.com/40604582/132090466-aebf0998-c0f8-4d42-a048-7360973e11e6.gif)